src/FontManagerImpl.h

```diff
@@ -0,0 +1,20 @@
+#include "FontDescriptor.h"
+#include "ResultSet.h"
+
+struct FontManagerImpl {
+  FontManagerImpl();
+  ~FontManagerImpl();
```

Comment:
I try to make all destructors on non-final classes either:
1. public and virtual
2. protected and non-virtual

to avoid someone down the road:
1. extending the class
2. deleting an object of the extended class via a pointer to the base class
3. :bomb: :boom: 

(looks like there are a few other areas we do this in this codebase)